src/WpfMath/Parsers/CommandParsers.cs

```diff
@@ -0,0 +1,66 @@
+using WpfMath.Atoms;
+using WpfMath.Exceptions;
+
+namespace WpfMath.Parsers
+{
+    /// <summary>A context that will be passed to the command parser.</summary>
+    internal class CommandContext
+    {
+        /// <summary>TeX formula parser that calls the command.</summary>
+        public TexFormulaParser Parser { get; }
+
+        /// <summary>Current formula.</summary>
+        public TexFormula Formula { get; }
+
+        /// <summary>Source of the current formula. Includes both command name and the arguments.</summary>
+        public SourceSpan FormulaSource { get; }
```

Comment:
It is not a "source of the current formula", i.e. it doesn't necessarily correlate with the `Formula` member. Let's better call it a `CommandSource` or something?

Reply:
`CommandSource` is fine.